src/shardus/shardus-types.ts

```diff
@@ -1257,6 +1261,10 @@ export interface ServerConfiguration {
     removeStuckTxsFromQueue2: boolean
     // how long to wait before removing the tx from the queue. Based on time since first vote seen
     stuckTxRemoveTime2: number
+    // whether to remove the tx from the queue if it stuck for X min. must be in consensing state
+    removeStuckTxsFromQueue3: boolean
+    // how long to wait before removing the tx from the queue. must be in consensing state
+    stuckTxRemoveTime3: number
```

Comment:
IMHO splitting feature flag to 2 parameters can add mess during support at a later stages.
for example, if stuckTxRemoveTime3 > 0 - feature is enabled, otherwise, disabled

Reply:
I like the two flags because operationally we usually think in terms of turning a feature on/off.  if we have to set the correct tuned time here I think that leaves more for error.